pkgs/applications/editors/emacs/elisp-packages/manual-packages/codeium/default.nix

```diff
@@ -1,9 +1,10 @@
 {
+  lib,
+  codeium,
   fetchFromGitHub,
   melpaBuild,
-  pkgs,
-  lib,
```

Comment:
Ideally, we should not move parameters around.

Reply:
`lib` is tipically the first parameter around Nixpkgs.

Further, `pkgs` was deleted.